protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -288,8 +288,9 @@ func nestedQueryParams(message *descriptor.Message, field *descriptor.Field, pre
 			}
 			if items != nil { // array
 				param.Items = &openapiItemsObject{
-					Type: "string",
-					Enum: listEnumNames(reg, enum),
+					schemaCore: schemaCore{Type: "string",
+						Enum: listEnumNames(reg, enum),
+					},
```

Comment:
Split this over multiple lines please
```suggestion
					schemaCore: schemaCore{
						Type: "string",
						Enum: listEnumNames(reg, enum),
					},
```